### PR TITLE
feat(technical setup): connector registration overlay

### PR DESCRIPTION
**Pull request by lavanya-bmw**
## Description

Addressed feedback for PR https://github.com/eclipse-tractusx/portal-frontend/pull/717

## Why

NA

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/694

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

**Comment by lavanya-bmw** [comment]
Changelog already added in https://github.com/eclipse-tractusx/portal-frontend/pull/717

**Review by oyo — APPROVED** [review]

File: src/assets/locales/de/main.json [hunk 1]
```diff
@@ -1307,7 +1307,7 @@
       }
     },
     "serviceMarketplace": {
-      "headline": "Purchase Service Request",
+      "headline": "Service-Abonnement",
```
Comment (jjeroch) [inline src/assets/locales/de/main.json:1310]: @lavanya-bmw  is it correct that the PR includes this change?

Comment (lavanya-bmw) [reply src/assets/locales/de/main.json:1310]: @jjeroch this was not part of this ticket but added this here since it's a small heading change

Comment (lavanya-bmw) [reply src/assets/locales/de/main.json:1310]: @jjeroch Please confirm If I need to raise a new PR for this change?

Comment (jjeroch) [reply src/assets/locales/de/main.json:1310]: we can proceed with the same

**Review by evegufy — APPROVED** [review]

**Pull request merged**